src/main/java/org/mate/bookstorespringboot/controller/mapper/BookMapper.java

```diff
@@ -20,4 +22,11 @@ default Book toEntityWithId(BookRequestDto bookRequestDto, Long id) {
         book.setId(id);
         return book;
     }
+
+    BookDtoWithoutCategoryIds toDtoWithoutCategoryIds(Book book);
+
+    @AfterMapping
+    default void setCategoryId(Book book, BookDto bookDto) {
+
```

Comment:
Should you add some realization?

Reply:
not really, forgot to deleted it before pushing 🫨